### ISSUE TITLE
For WritableVlan group attribute is marked as required

### ISSUE THREAD

**Issue by amhn** [netbox-community/netbox#10878]
### NetBox version

v3.3.7

### Python version

3.10

### Steps to Reproduce

Look at swagger.json. Required field for WritableVlan:
```
          "required" : [ 
              "group",
              "vid",
              "name"
            ],
```

Run the following POST request.
```
curl \
-H "Authorization: Token $NETBOX_TOKEN"\
-H "Content-type: application/json"\
http://localhost:8000/api/ipam/vlans/ -X POST -d "{\"vid\":53,\"name\":\"test\"}"
```

### Expected Behavior

According to swagger.json `group` is a required parameter. So this request should fail.

### Observed Behavior

Vlan is created.

This is actually the desired result. `group` should be dropped from the required parameters.

**Comment by amhn** [comment]
Same problem for group parameter of WritableContact. I can open another bug once this one is validated.

**Comment by jeremystretch** [comment]
I believe this is a bug in the inspector relating to the use of `unique_together` on the model. It's been resolved in the v3.4 beta branch, where we've replaced `unique_together` with UniqueConstraints globally.

**Comment by amhn** [comment]
Just looked at the API docs for 3.4 and it is fixed there.

Thank you.
